recipe/recipe.yaml

```diff
@@ -0,0 +1,82 @@
+context:
+  name: pyright
+  version: 1.1.386
+
+recipe:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+build:
+  number: 10
+
+source:
+  url: https://pypi.io/packages/source/${{ name[0] }}/${{ name }}/pyright-${{ version }}.tar.gz
+  sha256: 8e9975e34948ba5f8e07792a9c9d2bdceb2c6c0b61742b068d2229ca2bc4a9d9
+
+outputs:
+  - package:
+      name: ${{ name|lower }}-node
+    build:
+      skip: [ppc64le]
+      script:
+        file: build_node${{ '.bat' if win else '.sh' }}
+    requirements:
+      build:
+        - pnpm
+        - pnpm-licenses
+    about:
+      license: MIT
+      license_file: third-party-licenses.txt
+
+  - package:
+      name: ${{ name|lower }}
+    build:
+      noarch: python
+      script:
+        file: build_python${{ '.bat' if win else '.sh' }}
+      python:
+        entry_points:
+          - pyright=pyright.cli:entrypoint
+          - pyright-python=pyright.cli:entrypoint
+          - pyright-langserver=pyright.langserver:entrypoint
+          - pyright-python-langserver=pyright.langserver:entrypoint
+
+    requirements:
+      host:
+      - python
+      - setuptools
+      - pip
```

Comment:
you could alternatively to only building on linux-64 and making it a noarch build it as a non-noarch package. this would require `python` and `cross-python_osx-arm64` to be in the build requirements `if: build_platform != target_platform` though: https://github.com/conda-forge/pydantic-core-feedstock/blob/8453dd9c1b19e1fb2567175d73c3501446b2d01f/recipe/meta.yaml#L21-L22